digital_benchmark/facebook_benchmark/views.py

```diff
@@ -1,3 +1,32 @@
 from django.shortcuts import render
+from django.http import HttpResponse
+
+import facebook as fb
+
+version = '3.1'
 
 # Create your views here.
+def feed(request, access_token):
+    graph = fb.GraphAPI(access_token=access_token, version=version)
```

Comment:
Move this to templates

Reply:
and make this a class based view rather than a function based view

Reply:
class-based views or class-based generic views

Reply:
TODO: move to templates after creating models and storing data using models